setup.py

```diff
@@ -18,7 +18,7 @@
     "numpy",
     "openpyxl",
     "pandas",
-    "scipy",
+    "scipy<1.9.3",
```

Comment:
Is it ok that we are not pinning scipy in ert?

Reply:
Pyscal should be independent of ert in this regard. Ert should pin if it needs to.